packages/material-ui-lab/src/TimelineItemDot/TimelineItemDot.js

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { capitalize } from '@material-ui/core/utils';
+import { withStyles } from '@material-ui/core/styles';
+
+export const styles = (theme) => ({
+  /* Styles applied to the root element. */
+  root: {
+    display: 'flex',
+    alignSelf: 'baseline',
+    borderStyle: 'solid',
+    borderWidth: 2,
+    flex: 0,
+    padding: 4,
+    borderRadius: 999,
+    // marginRight: 'auto',
+    // marginLeft: 'auto',
+    top: 8,
+    position: 'relative',
+    boxShadow: theme.shadows[2],
+  },
+  /* Styles applied to the root element if `color="default"` and `variant="default"`. */
+  defaultDefault: {
+    borderColor: theme.palette.grey[300],
```

Comment:
It feels that `color="default"` (or `color="grey"` if we rename it) doesn't provide enough contrast (relative to its surface area) with the white background.
Maybe?

```diff
diff --git a/packages/material-ui-lab/src/TimelineItemDot/TimelineItemDot.js b/packages/material-ui-lab/src/TimelineItemDot/TimelineItemDot.js
index 9c1700f91..08141ef9e 100644
--- a/packages/material-ui-lab/src/TimelineItemDot/TimelineItemDot.js
+++ b/packages/material-ui-lab/src/TimelineItemDot/TimelineItemDot.js
@@ -18,16 +18,15 @@ export const styles = (theme) => ({
     // marginLeft: 'auto',
     top: 8,
     position: 'relative',
-    boxShadow: theme.shadows[2],
   },
   /* Styles applied to the root element if `color="default"` and `variant="default"`. */
   defaultDefault: {
-    borderColor: theme.palette.grey[300],
-    backgroundColor: theme.palette.grey[300],
+    borderColor: theme.palette.grey[400],
+    backgroundColor: theme.palette.grey[400],
   },
   /* Styles applied to the root element if `color="default"` and `variant="outlined"`. */
   outlinedDefalut: {
-    borderColor: theme.palette.grey[300],
+    borderColor: theme.palette.grey[400],
     backgroundColor: 'white', // TODO: fix color white
   },
   /* Styles applied when the component is passed children. */
```

<img width="191" alt="contrast" src="https://user-images.githubusercontent.com/3165635/83943036-11781880-a7f9-11ea-84a1-9e5d0f9f5713.png">

Reply:
In this direction, we would need to reverse the increment direction in dark mode.

Reply:
I've used `grey[500]` looks much better 👍